CRM/Report/Form/Activity.php

```diff
@@ -564,7 +564,8 @@ public function where($recordType = NULL) {
                 empty($this->_params['activity_type_id_value'])
               ) {
                 if (empty($this->_params['include_case_activities_value'])) {
-                  $this->activityTypes = CRM_Core_PseudoConstant::activityType(TRUE, FALSE, FALSE, 'label', TRUE);
+                  $componentId = CRM_Core_Component::getComponentID('CiviCase');
+                  $this->activityTypes = CRM_Core_OptionGroup::values('activity_type', FALSE, FALSE, FALSE, " AND v.component_id !={$componentId} OR v.component_id IS NULL");
```

Comment:
All of the test fails seem to be here. Because $componentId is null if civicase is enabled.

[CRM_Report_Form_ActivityTest.testLongCustomFieldNames](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/CRM_Report_Form_ActivityTest/testLongCustomFieldNames/)
    [CRM_Report_Form_ActivityTest.testTargetAddressFields](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/CRM_Report_Form_ActivityTest/testTargetAddressFields/)
    [api_v3_ReportTemplateTest.testActivityDetails](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/api_v3_ReportTemplateTest/testActivityDetails/)
    [api_v3_ReportTemplateTest.testActivityDetailsNullFilters](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/api_v3_ReportTemplateTest/testActivityDetailsNullFilters/)
    [api_v3_ReportTemplateTest.testActivityDetailsContactFilter](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/api_v3_ReportTemplateTest/testActivityDetailsContactFilter/)
    [api_v3_ReportTemplateTest.testReportTemplateSelectWhere with data set #2](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/api_v3_ReportTemplateTest/testReportTemplateSelectWhere_with_data_set__2/)
    [api_v3_ReportTemplateTest.testReportTemplateGetRowsAllReports with data set #2](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/api_v3_ReportTemplateTest/testReportTemplateGetRowsAllReports_with_data_set__2/)
    [api_v3_ReportTemplateTest.testReportTemplateGetRowsAllReportsACL with data set #2](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/api_v3_ReportTemplateTest/testReportTemplateGetRowsAllReportsACL_with_data_set__2/)
    [api_v3_ReportTemplateTest.testReportTemplateGetStatisticsAllReports with data set #2](https://test.civicrm.org/job/CiviCRM-Core-PR/49961/testReport/junit/(root)/api_v3_ReportTemplateTest/testReportTemplateGetStatisticsAllReports_with_data_set__2/)

Reply:
Thanks. I'll see if my update fixes that.